README.md

```diff
@@ -13,87 +16,38 @@ The GenLayer prototype consists of the following main components:
 
 ## Installation
 
-### Window One
```

Comment:
I think we still need to know how to do this without the CLI, you should add these instructions below, before the "separate steps" section otherwise those instructions are not possible